// @esri/calcite-components/dist/cjs/it-ebbf3674.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/dev/LICENSE.md for details.
 * v2.13.1
 */
'use strict';

const calciteInputTimePicker_calciteTimePicker_entry = require('./calcite-input-time-picker.calcite-time-picker-dbb07894.js');
require('./index-651a3e7f.js');
require('./form-a0aa32d4.js');
require('./dom-20c48ef3.js');
require('./guid-88d8b131.js');
require('./resources-dd12972d.js');
require('./interactive-e0d0d313.js');
require('./key-f102c715.js');
require('./label-baa45f8c.js');
require('./component-37e59b5c.js');
require('./loadable-228291fd.js');
require('./browser-6b9f2bff.js');
require('./locale-c34d44f4.js');
require('./observers-f1a1157c.js');
require('./focusTrapComponent-f086068b.js');
require('./config-4c6d0f86.js');
require('./math-c39b6d82.js');
require('./t9n-055820a7.js');
require('./Validation-cfecdd00.js');
require('./input-d0cdd44d.js');

// Italian [it]
var locale = {
  name: 'it',
  weekdays: 'domenica_lunedì_martedì_mercoledì_giovedì_venerdì_sabato'.split('_'),
  weekdaysShort: 'dom_lun_mar_mer_gio_ven_sab'.split('_'),
  weekdaysMin: 'do_lu_ma_me_gi_ve_sa'.split('_'),
  months: 'gennaio_febbraio_marzo_aprile_maggio_giugno_luglio_agosto_settembre_ottobre_novembre_dicembre'.split('_'),
  weekStart: 1,
  monthsShort: 'gen_feb_mar_apr_mag_giu_lug_ago_set_ott_nov_dic'.split('_'),
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'DD/MM/YYYY',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY HH:mm',
    LLLL: 'dddd D MMMM YYYY HH:mm'
  },
  relativeTime: {
    future: 'tra %s',
    past: '%s fa',
    s: 'qualche secondo',
    m: 'un minuto',
    mm: '%d minuti',
    h: 'un\' ora',
    hh: '%d ore',
    d: 'un giorno',
    dd: '%d giorni',
    M: 'un mese',
    MM: '%d mesi',
    y: 'un anno',
    yy: '%d anni'
  },
  ordinal: function ordinal(n) {
    return n + "\xBA";
  }
};
calciteInputTimePicker_calciteTimePicker_entry.dayjs.locale(locale, null, true);

exports['default'] = locale;
