// @esri/calcite-components/dist/esm/es-653c88d9.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/dev/LICENSE.md for details.
 * v2.13.1
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-695e107e.js';
import './index-ca4a9906.js';
import './form-7c2a4548.js';
import './dom-656a3e0e.js';
import './guid-d335a5f3.js';
import './resources-192b6915.js';
import './interactive-ca1c0889.js';
import './key-a4b582ad.js';
import './label-fd307fdd.js';
import './component-e4499279.js';
import './loadable-27ff99ad.js';
import './browser-6b10ddb7.js';
import './locale-14b6fa72.js';
import './observers-abdc7046.js';
import './focusTrapComponent-e5960968.js';
import './config-56174dc6.js';
import './math-32c93c95.js';
import './t9n-7f1cc36b.js';
import './Validation-45e07a84.js';
import './input-c3865ac8.js';

// Spanish [es]
var locale = {
  name: 'es',
  monthsShort: 'ene_feb_mar_abr_may_jun_jul_ago_sep_oct_nov_dic'.split('_'),
  weekdays: 'domingo_lunes_martes_miércoles_jueves_viernes_sábado'.split('_'),
  weekdaysShort: 'dom._lun._mar._mié._jue._vie._sáb.'.split('_'),
  weekdaysMin: 'do_lu_ma_mi_ju_vi_sá'.split('_'),
  months: 'enero_febrero_marzo_abril_mayo_junio_julio_agosto_septiembre_octubre_noviembre_diciembre'.split('_'),
  weekStart: 1,
  formats: {
    LT: 'H:mm',
    LTS: 'H:mm:ss',
    L: 'DD/MM/YYYY',
    LL: 'D [de] MMMM [de] YYYY',
    LLL: 'D [de] MMMM [de] YYYY H:mm',
    LLLL: 'dddd, D [de] MMMM [de] YYYY H:mm'
  },
  relativeTime: {
    future: 'en %s',
    past: 'hace %s',
    s: 'unos segundos',
    m: 'un minuto',
    mm: '%d minutos',
    h: 'una hora',
    hh: '%d horas',
    d: 'un día',
    dd: '%d días',
    M: 'un mes',
    MM: '%d meses',
    y: 'un año',
    yy: '%d años'
  },
  ordinal: function ordinal(n) {
    return n + "\xBA";
  }
};
dayjs.locale(locale, null, true);

export { locale as default };
