// @esri/calcite-components/dist/esm/nb-1d751dfd.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/dev/LICENSE.md for details.
 * v2.13.1
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-695e107e.js';
import './index-ca4a9906.js';
import './form-7c2a4548.js';
import './dom-656a3e0e.js';
import './guid-d335a5f3.js';
import './resources-192b6915.js';
import './interactive-ca1c0889.js';
import './key-a4b582ad.js';
import './label-fd307fdd.js';
import './component-e4499279.js';
import './loadable-27ff99ad.js';
import './browser-6b10ddb7.js';
import './locale-14b6fa72.js';
import './observers-abdc7046.js';
import './focusTrapComponent-e5960968.js';
import './config-56174dc6.js';
import './math-32c93c95.js';
import './t9n-7f1cc36b.js';
import './Validation-45e07a84.js';
import './input-c3865ac8.js';

// Norwegian Bokmål [nb]
var locale = {
  name: 'nb',
  weekdays: 'søndag_mandag_tirsdag_onsdag_torsdag_fredag_lørdag'.split('_'),
  weekdaysShort: 'sø._ma._ti._on._to._fr._lø.'.split('_'),
  weekdaysMin: 'sø_ma_ti_on_to_fr_lø'.split('_'),
  months: 'januar_februar_mars_april_mai_juni_juli_august_september_oktober_november_desember'.split('_'),
  monthsShort: 'jan._feb._mars_april_mai_juni_juli_aug._sep._okt._nov._des.'.split('_'),
  ordinal: function ordinal(n) {
    return n + ".";
  },
  weekStart: 1,
  yearStart: 4,
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'DD.MM.YYYY',
    LL: 'D. MMMM YYYY',
    LLL: 'D. MMMM YYYY [kl.] HH:mm',
    LLLL: 'dddd D. MMMM YYYY [kl.] HH:mm'
  },
  relativeTime: {
    future: 'om %s',
    past: '%s siden',
    s: 'noen sekunder',
    m: 'ett minutt',
    mm: '%d minutter',
    h: 'en time',
    hh: '%d timer',
    d: 'en dag',
    dd: '%d dager',
    M: 'en måned',
    MM: '%d måneder',
    y: 'ett år',
    yy: '%d år'
  }
};
dayjs.locale(locale, null, true);

export { locale as default };
