// @esri/calcite-components/dist/esm/sv-80e3d9df.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/dev/LICENSE.md for details.
 * v2.13.1
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-695e107e.js';
import './index-ca4a9906.js';
import './form-7c2a4548.js';
import './dom-656a3e0e.js';
import './guid-d335a5f3.js';
import './resources-192b6915.js';
import './interactive-ca1c0889.js';
import './key-a4b582ad.js';
import './label-fd307fdd.js';
import './component-e4499279.js';
import './loadable-27ff99ad.js';
import './browser-6b10ddb7.js';
import './locale-14b6fa72.js';
import './observers-abdc7046.js';
import './focusTrapComponent-e5960968.js';
import './config-56174dc6.js';
import './math-32c93c95.js';
import './t9n-7f1cc36b.js';
import './Validation-45e07a84.js';
import './input-c3865ac8.js';

// Swedish [sv]
var locale = {
  name: 'sv',
  weekdays: 'söndag_måndag_tisdag_onsdag_torsdag_fredag_lördag'.split('_'),
  weekdaysShort: 'sön_mån_tis_ons_tor_fre_lör'.split('_'),
  weekdaysMin: 'sö_må_ti_on_to_fr_lö'.split('_'),
  months: 'januari_februari_mars_april_maj_juni_juli_augusti_september_oktober_november_december'.split('_'),
  monthsShort: 'jan_feb_mar_apr_maj_jun_jul_aug_sep_okt_nov_dec'.split('_'),
  weekStart: 1,
  yearStart: 4,
  ordinal: function ordinal(n) {
    var b = n % 10;
    var o = b === 1 || b === 2 ? 'a' : 'e';
    return "[" + n + o + "]";
  },
  formats: {
    LT: 'HH:mm',
    LTS: 'HH:mm:ss',
    L: 'YYYY-MM-DD',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY [kl.] HH:mm',
    LLLL: 'dddd D MMMM YYYY [kl.] HH:mm',
    lll: 'D MMM YYYY HH:mm',
    llll: 'ddd D MMM YYYY HH:mm'
  },
  relativeTime: {
    future: 'om %s',
    past: 'för %s sedan',
    s: 'några sekunder',
    m: 'en minut',
    mm: '%d minuter',
    h: 'en timme',
    hh: '%d timmar',
    d: 'en dag',
    dd: '%d dagar',
    M: 'en månad',
    MM: '%d månader',
    y: 'ett år',
    yy: '%d år'
  }
};
dayjs.locale(locale, null, true);

export { locale as default };
